ngraph/core/src/op/if.cpp

```diff
@@ -18,8 +18,7 @@
 using namespace std;
 using namespace ngraph;
 
-OPENVINO_RTTI_DEFINITION(ngraph::op::v8::If, "If", 8, MultiSubGraphOp);
-
+BWDCMP_RTTI_DEFINITION(op::v8::If);
```

Comment:
For new operations, I don't want to introduce backward compatibility code. We need to have this macro only to avoid situations if user uses`...::type_info` in the application code, but he cannot use such code for operations which haven't been released yet.

Reply:
Seems you can remove it from here and for other `v8` ops, take a look also at `v8::Gather`, `v8::DeformableConvolution`,    `v8::MatrixNms`

Reply:
Or do you mean `BWDCMP_RTTI_DEFINITION` is not needed only for new operations without any previous versions?

Reply:
Yes, I need to have `BWDCMP_RTTI_DEFINITION` if any operation in the inheritance structure defines `type_info` static member.